runner/runners/invoke.go

```diff
@@ -131,17 +134,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
```

Comment:
My preferred solution is to set $TMPDIR to the desired location in our aurora deploy. The code shouldn't have to know that there are system limitations. WDYT?